grpc/nodejs/package.json

```diff
@@ -10,12 +10,10 @@
     "url": "https://github.com/vaticle/typedb-protocol"
   },
   "dependencies": {
-    "@grpc/grpc-js": "1.5.0"
   },
   "devDependencies": {
-    "@bazel/typescript": "2.3.1",
-    "grpc-tools": "1.11.2",
-    "grpc_tools_node_protoc_ts": "5.3.2",
-    "typescript": "3.9.7"
+    "protoc-gen-ts": "0.8.6",
```

Comment:
@lolski The guy who provides this library asks for donations to help support the development, I wonder if we should try to think about a policy around this kind of thing? He's saved us so much time by providing it...
cc @alexjpwalker  @benjaminasmall